### PR TITLE
Fix a broken external link

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Fix a broken external link.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-documentation-guide.md [hunk 1]
```diff
@@ -22,5 +22,4 @@ aliases: ['/docs-cn/dev/tiup/tiup-documentation-guide/']
 
 ## TiUP 资源
 
-- [AskTUG TiUP](https://asktug.com/c/qa/set-up-deploy)：AskTUG.com 部署板块
```
Comment (yikeke) [inline tiup/tiup-documentation-guide.md:25]: - [AskTUG TiUP 主题](https://asktug.com/tags/tiup)

File: tiup/tiup-documentation-guide.md [hunk 2]
```diff
@@ -22,5 +22,5 @@ aliases: ['/docs-cn/dev/tiup/tiup-documentation-guide/']
...
+- [AskTUG TiUP](https://asktug.com/c/administration)：AskTUG.com 部署板块
```
Comment (TomShawn) [inline tiup/tiup-documentation-guide.md:25]:
```suggestion
- [AskTUG TiUP](https://asktug.com/tags/tiup)：AskTUG.com 部署板块
```

File: tiup/tiup-documentation-guide.md [hunk 3]
```diff
@@ -22,5 +22,5 @@ aliases: ['/docs-cn/dev/tiup/tiup-documentation-guide/']
 
 ## TiUP 资源
 
-- [AskTUG TiUP](https://asktug.com/c/qa/set-up-deploy)：AskTUG.com 部署板块
+- [AskTUG TiUP](https://asktug.com/tags/tiup)：AskTUG.com 部署板块
```
Comment (yikeke) [inline tiup/tiup-documentation-guide.md:25]:
```suggestion
- [AskTUG TiUP 主题](https://asktug.com/tags/tiup)
```

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**